chat-plugins/uno.js

```diff
@@ -163,39 +217,47 @@ class UNOgame extends Rooms.RoomGame {
 		this.players[user.userid].userid = user.userid;
 		if (this.awaitUno && this.awaitUno === oldUserid) this.awaitUno = user.userid;
 
-		if (this.currentPlayer && this.currentPlayer === oldUserid) this.currentPlayer = user.userid;
+		if (this.currentPlayerid && this.currentPlayerid === oldUserid) this.currentPlayerid = user.userid;
```

Comment:
You shouldnt need to check if currentPlayerid is falsey, they should both be strings and a simple compare check should work (which your already doing).

Reply:
Yeah this was when currentPlayer still defaulted to null and i forgot to remove null checks